src/annotator/hypothesis-injector.js

```diff
@@ -41,33 +41,32 @@ export class HypothesisInjector {
   }
 
   /**
-   * Inject Hypothesis client into a newly-discovered iframe.
+   * Inject Hypothesis client into a newly-discovered iframe. The DOM's iframe
+   * is guaranteed to be accessible and ready.
```

Comment:
The second sentence here implies that this is a guarantee made by _this method_. What you presumably mean however is that this is a precondition of this method (ie. that the iframe must be "accessible" and "ready" before this method is called. It would be helpful to clarify briefly what "accessible" and "ready" mean here).

Reply:
Yes, that's what I meant. `FrameObserver._addFrame` wrapper around `HypothesisInjector._addHypothesis` guarantees the precondition (DOM ready) is fulfilled.